### ISSUE TITLE
backtrace-node crashes application on Windows 10 when there is no network connection

### ISSUE THREAD

**Issue by alexanderturinske** [backtrace-labs/backtrace-node#16]
OS: Windows 10
backtrace-node version: 0.7.4
Code
```
bt.initialize({
    endpoint: BACKTRACE_URL,
    token: BACKTRACE_TOKEN,
    allowMultipleUncaughtExceptionListeners: true,
    handlePromises: true,
    attributes: {
        ver: devDependencies.electron,
        platform: os.platform(),
        process_type: 'main',
        application: 'Electron',
        _version: app.getVersion(),
        _productName: app.getName(),
        _companyName: productName,
        user: (os.userInfo()).username,
    },
});
```

1. Build the .exe (using `electron-packager`)
2. Open the `Command Prompt`
3. Navigate to the directory with the `.exe` is in
4. Open the `.exe` via `Command Prompt` (i.e. `$ YourApp.exe`)
     The errors should now output to the `Command Prompt`; hopefully they are useful. Mine gave:
```
Error: callback called twice
    at onCb (C:\Users\Alexander\path\to\app\dist\PIX-win32-x64\resources\app.asar\node_modules\pend\index.js:36:23)
    at C:\Users\Alexander\path\to\app\dist\PIX-win32-x64\resources\app.asar\node_modules\backtrace-node\lib\send_report.js:280:9
    at ChildProcess.<anonymous> (C:\Users\Alexander\path\to\app\dist\PIX-win32-x64\resources\app.asar\node_modules\backtrace-node\lib\send_report.js:302:7)
    at ChildProcess.emit (events.js:182:13)
    at maybeClose (internal/child_process.js:961:16)
    at Socket.stream.socket.on (internal/child_process.js:380:11)
    at Socket.emit (events.js:182:13)
    at Pipe._handle.close [as _onclose] (net.js:596:12)
```

Relates to [electron/electron #6437](https://github.com/electron/electron/issues/6437)

**Comment by sbahra** [comment]
Thanks for the report! Looking into this for you.

**Comment by mbreaux** [comment]
Hi @alexanderturinske, apologies for the delay.  I've been trying to reproduce this on my test Windows 10 machine and I've had no luck so far.

I figured we could compare out setups and see what's different in an effort to narrow this down.

I'm using Node.js 10.2.0, Electron 3.0.4 on Windows 10 (10.0.15063), backtrace-node 0.7.4.  I'm adding the bt.initialize code at the beginning of main.js of the electron-quick-start app, and building that with electron-packager.  I'm running this at the command prompt with all network adapters disabled, and not getting the "callback called twice" error.

Are your node and electron versions up to date?  I'd like to rule out that I'm running a later version of one of these components.

**Comment by sbahra** [comment]
@alexanderturinske Hi there. Could you help us get some more information? We are having issues recreating this.

**Comment by alexanderturinske** [comment]
Sorry for not getting back to you in a timely manner.

More information:
`node: v8.12`,
`Electron: https://github.com/castlabs/electron0releases#v3.0.2-wvvmp`
`Windows: 10, version 1803, OS build: 17134.345`

Re-reading what I wrote, I am 🤦‍♀️  because I left out a very crucial part; I was on a proxy while I was doing this that was not letting any outgoing requests occur. Thus, I was able to recreate the issue off of a proxy by simply removing my computer from internet access (i.e. go on airplane mode). If I turn off internet to my machine, I get the same error.

I apologize immensely for leaving that out.
